app/src/utils/validate-item.ts

```diff
@@ -10,11 +10,12 @@ export function validateItem(
 	fields: Field[],
 	isNew: boolean,
 	includeCustomValidations = false,
+	currentVersion: ContentVersion | null,
```

Comment:
The default should be `null` here.

```suggestion
	currentVersion: ContentVersion | null = null,
```